### ISSUE TITLE
teleport players to the ground, instead of on top of Region Posts

### ISSUE THREAD

**Issue by spooner01** [MLG-Fortress/PopulationDensity#32]
Hey there! I love this plugin, but you probably hear that all the time. Im not super good at deciphering the backend of plugin structure (outside of configs) so I was hoping you might be able to point me in the right direction on something. 

when teleporting to a region post players spawn on top of the post. I cant find a setting to adjust in the config to change how close to the post a player will spawn. Is it possible to let a player spawn within 1-2 blocks of the post? Maybe what I'm looking for is too complicated....

TL;DR
i would like to teleport players to the region posts but not on top of the post. It would be super cool if players would land near the post, on the ground, anywhere in the 5x5 area that is a region post.

**Comment by RoboMWM** [comment]
Basically https://github.com/BigScary/PopulationDensity/pull/3

However I think it's best to leave this to an addon regarding something like this (especially considering #29)

But if added directly, it must be configurable.
